openfl/component/director/director.py

```diff
@@ -64,20 +64,25 @@ async def set_new_experiment(self, *, experiment_name: str, sender_name: str,
         """Set new experiment."""
         # TODO: save to file
         self.experiment_data[experiment_name] = data
+        experiment_work_dir = Path(f'{os.getcwd()}/{sender_name}/{experiment_name}')
```

Comment:
We discussed making timestamps-names for experiment working directories, is it a better idea?